templates/download/cloud/index.html

```diff
@@ -10,191 +10,123 @@
-        <p>
-          Ubuntu builds optimised and certified server images for partners like Amazon AWS, Microsoft Azure, Google Cloud Platform, IBM Cloud and Oracle. In addition, Canonical offers Ubuntu Pro, enterprise-grade commercial support, on these images.
-        </p>
-      </div>
-      <div class="col-4 u-align--center u-vertically-center u-hide--medium u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/f0028afb-ubuntu-cloud.svg",
-                alt="ubuntu cloud illustration",
-                width="303",
-                height="198",
-                hi_def=True,) | safe
-        }}
-      </div>
+<section class="p-section--hero">
+  <div class="row--50-50">
+    <div class="col p-section--shallow">
+      <h1>Optimised Ubuntu<br /> on public clouds</h1>
     </div>
-  </section>
-
-  {% include "/shared/_ubuntu_pro_on_public_cloud.html" %}
-
-  <section class="p-strip is-deep is-bordered">
-    <div class="row">
-      <div class="col-10">
-        <h2>Use Ubuntu in the cloud</h2>
-        <p>
-          Here are some shortcuts to get you up and running with Ubuntu on various public clouds. You can also use our <a href="https://cloud-images.ubuntu.com/locator/">Ubuntu Cloud image finder</a> to search for the ids of specific Ubuntu images on various public clouds by location, architecture, release and more.
-        </p>
-      </div>
+    <div class="col">
+      <p>Ubuntu builds optimised and certified images of <a href="https://ubuntu.com/server">Ubuntu Server</a> for partners like Amazon AWS, Microsoft Azure, Google Cloud, IBM Cloud and Oracle Cloud.</p>
+      <ul class="p-list--divided">
+        <li class="p-list__item is-ticked">5 years of maintenance and security updates with Ubuntu LTS</li>
+        <li class="p-list__item is-ticked">Free from licence fees, regardless of how many images you run</li>
+      </ul>
     </div>
-    <div class="row u-equal-height">
-      <div class="p-card col-4">
-        <div class="p-card__content">
-          <h4>
-            <a class="u-align--center u-vertically-center"
-               style="min-height:10rem;
-                      display: flex"
-               href="https://cloud-images.ubuntu.com/locator/ec2/">
-              {{ image(url="https://assets.ubuntu.com/v1/a82add58-profile-aws.svg",
-                            alt="Amazon Web Services logo",
-                            width="84",
-                            height="50",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"style": "align-self: center;max-height: 10rem; max-width:10rem;"},) | safe
-              }}
+  </div>
+  <div class="row">
+    <div class="col-start-large-4 col-9 ">
+      <hr class="is-muted" />
+      <div class="row">
+        <div class="col-3 col-medium-3">
+          <h2>
+            <a href="/pro">
+              {{
+                image(
```

Comment:
did you run formatting? it was supposed to look different by djLint. Maybe try putting the first parameter on the same line with `image(`